tests/conftest.py

```diff
@@ -25,25 +25,9 @@
 """Common pytest fixtures and plugins."""
 
 import json
-from copy import deepcopy
 from os.path import dirname, join
 
-import mock
 import pytest
-from invenio_circulation.proxies import current_circulation
```

Comment:
I saw it in an other PR. Please check with zub.

Reply:
`current_circulation` is not used in `tests/conftest.py`